### PR TITLE
Add ranking numbers

### PR DESCRIPTION
**Pull request by TwelveNights**

### CODE REVIEW

File: client/Main.css [hunk 1]
```diff
@@ -16,8 +16,8 @@
 }
 
 .player-entry {
-	
-	
+
+
```
Comment (Futochu) [inline client/Main.css]: stop it

**Comment by TwelveNights** [comment]
Already fixed sort order to elo.

File: imports/ui/Body.js [hunk 2]
```diff
@@ -1,40 +1,48 @@
 import { Template } from 'meteor/templating';
-
 import { Players } from '../api/Players.js';
+import { Constants } from '../api/Constants.js';
 
 import './About.html';
-import './Body.html';
 import './Home.html';
+import './Body.html';
```
Comment (Futochu) [inline imports/ui/Body.js]: Why is this reordered?  I'd prefer if imports were in alphabetical order (I know they weren't before but I also tried to group similar imports)

File: imports/ui/Body.js [hunk 3]
```diff
@@ -1,40 +1,48 @@
...
 import './JapaneseNewGame.js';
 
-Template.JapaneseRanking.helpers({
-	japanesePlayers() {
-		return Players.find({}, {sort: { japaneseElo: -1}});
-	},
-});
+Template.registerHelper('toObj', (args) => {
+	return args.hash;
+})
```
Comment (Futochu) [inline imports/ui/Body.js]: Style: End with a semicolon please

File: client/Main.css (hunk 1 above)
Comment (TwelveNights) [inline client/Main.css]: What, removing whitespaces?

Comment (Futochu) [inline client/Main.css]: For some reason it looked like you added whitespace

Comment (TwelveNights) [inline client/Main.css]: It removes the bolded out spaces, which are tabspaces and replaces them with newlines

File: imports/ui/Body.js (hunk 2 above)
Comment (TwelveNights) [inline imports/ui/Body.js]: Reordered to alphabetical.

File: imports/ui/Body.js (hunk 3 above)
Comment (TwelveNights) [inline imports/ui/Body.js]: Done

File: imports/ui/ranking/JapaneseRanking.html [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<template name="JapaneseRanking">
+	{{ > Ranking 
+		name="Japanese Majong ELO Ranking"
+		sortBy=(toObj japaneseElo=-1)
+		format="jpn"
```
Comment (Futochu) [inline imports/ui/ranking/JapaneseRanking.html]: Can you set this to a function returning Constants.GAME_TYPE.JAPANESE

File: imports/ui/ranking/HongKongRanking.html [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<template name="HongKongRanking">
+	{{ > Ranking 
+		name="Hong Kong Mahjong ELO Rankings"
+		sortBy=(toObj hongKongElo=-1)
+		format="hkg"
```
Comment (Futochu) [inline imports/ui/ranking/HongKongRanking.html]: Can you set this to a function returning Constants.GAME_TYPE.HONG_KONG

File: imports/ui/ranking/HongKongRanking.html [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<template name="HongKongRanking">
+	{{ > Ranking 
+		name="Hong Kong Mahjong ELO Rankings"
```
Comment (Futochu) [inline imports/ui/ranking/HongKongRanking.html]: Maybe this title and the Japanese title should be constants or assets.  Thoughts?

File: imports/ui/ranking/JapaneseRanking.html (hunk 4 above)
Comment (TwelveNights) [inline imports/ui/ranking/JapaneseRanking.html]: I've moved GAME_TYPE.JAPANESE into the html namespace.

File: imports/ui/ranking/HongKongRanking.html (hunk 6 above)
Comment (TwelveNights) [inline imports/ui/ranking/HongKongRanking.html]: Good idea. We can actually retrieve the name of the league from the format.

File: imports/ui/ranking/HongKongRanking.html (hunk 5 above)
Comment (TwelveNights) [inline imports/ui/ranking/HongKongRanking.html]: No problem.

File: imports/ui/Body.js [hunk 7]
```diff
@@ -1,40 +1,66 @@
...
+			"leagueName": leagueName,
+			"elo": elo.toFixed(3),
+			"rank": this.rank ? ++this.rank : this.rank = 1
+		};
 	},
-});
+	getName(format) {
+		switch (format) {
+			case Constants.GAME_TYPE.JAPANESE:
+				league = "Japanese";
```
Comment (Futochu) [inline imports/ui/Body.js]: Without var/lib this will create a global variable

**Comment by Futochu** [comment]
Oh wait I already merged this

**Pull request merged**